app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -334,6 +338,8 @@ class ChatActivity : AppCompatActivity(), IChatView {
 
     //Replies user with Speech
     override fun voiceReply(reply: String, language: String) {
+        searchChat.visibility = View.VISIBLE
+        fabsetting.visibility = View.VISIBLE
```

Comment:
And what about it?

Reply:
> And what about it?

When we open up sttfragment, and give some voice commands, the chat activity is not again restarted. It simply calls the function voiceReply. So, it was necessary to make the visibility here